mlir/test/tblgen-to-irdl/TestDialect.td

```diff
@@ -41,9 +40,38 @@ def Test_AnyOp : Test_Op<"any"> {
 // CHECK-LABEL: irdl.operation @any {
 // CHECK-NEXT:    %[[v0:[^ ]*]] = irdl.any
 // CHECK-NEXT:    irdl.operands(%[[v0]])
-// CHECK-NEXT:    irdl.results()
 // CHECK-NEXT:  }
 
+// Check confined types are converted correctly.
+def Test_ConfinedOp : Test_Op<"confined"> {
+  let arguments = (ins ConfinedType<I32, [IntNonNegative.predicate]>:$confined,
```

Comment:
Can `IntNonNegative.predicate` be replaced with a string, so we can more easily check it in `CHECK-NEXT`.

Reply:
I've changed the example to represent vector types and tensor types, using CPreds